plugins/cybereason/help.md

```diff
@@ -45,6 +45,218 @@ Example input:
 
 ### Actions
 
+#### Archive Sensor
+
+This action is used to archive sensor.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|argument|string|None|True|The reason for archiving the sensor or sensors|None|Sensors are no longer in use|
+|sensor_ids|[]string|None|True|The unique identifier of the machine(s) you wish to perform the operation on|None|["58ae74fae4b06dca39c1d4bc:PYLUMCLIENT_ORG1-PROD_WINTEST-PC_005056A104F9"]|
+
+Example input:
+
+```
+{
+  "argument": "Sensors are no longer in use",
+  "sensor_ids": [
+    "58ae74fae4b06dca39c1d4bc:PYLUMCLIENT_ORG1-PROD_WINTEST-PC_005056A104F9"
+  ]
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|archive_sensor_response|archiveSensorResponse|True|Archive sensor response|
+
+Example output:
+
+```
+{
```

Comment:
Pretty print please 
https://jsonformatter.org/json-pretty-print <- This is really useful